test/service/payment.service.spec.ts

```diff
@@ -60,13 +60,6 @@ describe('Payment Service test suite', () => {
     expect(response).toEqual(CONFIRMATION_URL);
   });
 
-  test(`startPaymentsSession() should return ${CONFIRMATION_URL} if FEATURE_FLAG_ENABLE_PAYMENT_16052022 is 'false', '0', 'off' or ""`, async () => {
```

Comment:
Well done, just minor change, we do not need the `mockIsActiveFeature`, these line of code can be removed
```
jest.mock("../../src/utils/feature.flag" );
...
import { isActiveFeature } from "../../src/utils/feature.flag";
...
const mockIsActiveFeature = isActiveFeature as jest.Mock;
mockIsActiveFeature.mockImplementation( () => true );
...
```